plugin/trino-influxdb/src/main/java/io/trino/plugin/influxdb/optimize/InfluxLimitManager.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.influxdb.optimize;
+
+import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.connector.ConnectorTableHandle;
+
+public class InfluxLimitManager
+{
+    public String buildLimit(ConnectorSession session, ConnectorTableHandle handle, long limit)
+    {
+        return " LIMIT " + limit;
+    }
+
+    public String buildSLimit(ConnectorSession session, ConnectorTableHandle handle, long slimit)
+    {
+        return " SLIMIT " + slimit;
```

Comment:
Could you add a comment that explains `SLIMIT`?

Reply:
It use to limit the number of series to return from the specified measurement, only works on aggregation pushdown. I will delete them in this PR according to your suggestions.